src/calliope/preprocess/model_math.py

```diff
@@ -58,79 +59,116 @@ def __iter__(self):
         for key in self.ATTRS_TO_SAVE:
             yield key, deepcopy(getattr(self, key))
 
-    def _init_from_list(
-        self, math_to_add: list[str], model_def_path: str | Path | None = None
-    ):
-        """Load math definition from a list of files.
+    def __repr__(self) -> str:
+        """Custom string representation of class."""
+        return f"""Calliope math definition dictionary with:
+    {len(self.data["variables"])} decision variable(s)
+    {len(self.data["global_expressions"])} global expression(s)
+    {len(self.data["constraints"])} constraint(s)
+    {len(self.data["piecewise_constraints"])} piecewise constraint(s)
+    {len(self.data["objectives"])} objective(s)
+        """
```

Comment:
Incorrect tabs trigger a `mypy` error.
```suggestion
    def __repr__(self) -> str:
        """Custom string representation of class."""
        return f"""
        Calliope math definition dictionary with:
        {len(self.data["variables"])} decision variable(s)
        {len(self.data["global_expressions"])} global expression(s)
        {len(self.data["constraints"])} constraint(s)
        {len(self.data["piecewise_constraints"])} piecewise constraint(s)
        {len(self.data["objectives"])} objective(s)
        """
```